.github/workflows/ci.yml

```diff
@@ -34,13 +34,13 @@ jobs:
               elixir: 1.13.3
               otp: 23.3
           - pair:
-              elixir: 1.13.3
-              otp: 24.3
+              elixir: 1.13.4
```

Comment:
Could you please keep the 1.13/OTP24 pair and add an additional 1.13/OTP25 pair instead. For the latest Elixir version, I prefer to have all combinations tested. You can also bump the Elixir version to 1.13.4 in the other pairs.